examples/README.md

```diff
@@ -0,0 +1,38 @@
+# examples
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Requirements
+
+No requirements.
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| <a name="provider_aws"></a> [aws](#provider\_aws) | 3.74.3 |
```

Comment:
Since the module works well with versions 3.x and higher including 4.x, do we need to lock it with 3.x?

Reply:
It is not explicitly locked because it accepts any version above 3.5 @JohnNMwangi